compiler+runtime/src/cpp/jank/util/escape.cpp

```diff
@@ -5,6 +5,18 @@
 
 namespace jank::util
 {
+  /* Converts escape sequences (2 characters) to their mapped (single) character:
+   *  \n => '\n'
+   *  \t => '\t'
+   *  \r => '\r'
+   *  \\ => '\\'
+   *  \" => '"'
+   *  \a => '\a'
+   *  \v => '\v'
+   *  \? => '?'
+   *  \f => '\f'
+   *  \b => '\b'
```

Comment:
(applies to escape and unescape)

I understand the goal of documenting these fns to avoid confusion. Let's not exhaustively duplicate the entire mapping into the comments, though; that's very clearly seen in the `switch` statement. We can give an example of '\n' becoming \n and vice versa.